### PR TITLE
Poc docker angular

### PR DESCRIPTION
**Pull request by esion**
TODO: 
* [ ] nodemon has been remove from caliopen-dev start script and should be done here : https://github.com/CaliOpen/caliopen-dev/pull/36/files

### CODE REVIEW

File: development.yaml [hunk 1]
```diff
@@ -2,8 +2,10 @@
 env: development
 frontend:
   rootPath: "/app/"
-  path: ../frontend
-  cssFiles: []
+  path: "/srv/caliopen/web-client-ng"
```
Comment (gdchamal) [inline development.yaml]: relative path not absolute hence will work only on poc vm

**Comment by gdchamal** [comment]
LGTM I merge

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,13 @@
+FROM node:0.12
+
+RUN apt-get update && \
+    apt-get upgrade
+
+ADD ./ /srv/caliopen/web
+WORKDIR /srv/caliopen/web/
+
+RUN npm install
```
Comment (ziir) [inline Dockerfile]: You probably want to `RUN npm i npm@latest-2 -g` before install

File: app/views/base.html [hunk 3]
```diff
@@ -4,6 +4,7 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <title>CaliOpen</title>
+    <link rel="shortcut icon" sizes="16x16 32x32 48x48 64x64" href="/app/images/favicon.ico">
```
Comment (ziir) [inline app/views/base.html]: Favicon should be served at server's root : `http://www.caliopen.web/favicon.ico`.

Maybe the express favicon middleware can help you with that

File: Dockerfile (hunk 2 above)
Comment (esion) [inline Dockerfile]: I don't get it. 

It run with packaged npm from base image. And a day we should upgrade to node lts and its npm

File: app/views/base.html (hunk 3 above)
Comment (esion) [inline app/views/base.html]: :+1: you right I add a trello card

File: Dockerfile (hunk 2 above)
Comment (ziir) [inline Dockerfile]: Ok :+1:

**Pull request merged**